assets/i18n/en.json

```diff
@@ -773,8 +773,13 @@
     "I18N_TOPNAV_ADMIN_PAGE": "Admin Page",
     "I18N_TOPNAV_BLOG": "Blog",
     "I18N_TOPNAV_BLOG_DASHBOARD": "Blog Dashboard",
-    "I18N_TOPNAV_CLASSROOM": "Classroom",
-    "I18N_TOPNAV_CLASSROOM_BASIC_MATHS": "Basic Mathematics",
+    "I18N_TOPNAV_CLASSROOM": "Learn",
+    "I18N_TOPNAV_CLASSROOM_BASIC_MATHS": "Math Foundations",
+    "I18N_TOPNAV_CLASSROOM_DESCRIPTION_1": "Beginner friendly lessons to help you get started in math.",
+    "I18N_TOPNAV_CLASSROOM_DESCRIPTION_2": "Additional resources made by the community to help you learn more.",
```

Comment:
Its not classroom description, the link takes us to the community library. Classroom and library are different.